lib/services/microsoft_teams.rb

```diff
@@ -189,8 +189,9 @@ def facts
 
     payload.audit.changes.each do |obj|
       obj[title_key] = obj.delete("field_name")
+
       # convert textual fields to markdown that Microsoft can display
-      if obj["value"].to_s.include?("</span>")
+      if obj["value"].to_s.match(/<\/?(span|div)>/)
```

Comment:
It would be nice if you left a comment with some examples for what we are trying to match here.